src/gui/trackerlogin.cpp

```diff
@@ -43,7 +43,7 @@ trackerLogin::trackerLogin(QWidget *parent, BitTorrent::TorrentHandle *const tor
 
   buttonBox->button(QDialogButtonBox::Ok)->setText(tr("Log in"));
 
-  login_logo->setPixmap(QPixmap(QString::fromUtf8(":/icons/qbt-theme/encrypted.png")));
+  login_logo->setPixmap(QPixmap(":/icons/qbt-theme/encrypted.png"));
```

Comment:
`QLatin1String`?

Reply:
I suggest use QLatin1String because QString then replaces UTF-8 → UTF-16 conversion with Latin1 → UTF-16 quickie. This is not important for rarely used short literals, of course, so here I propose to use it for consistency and to highlight that the literal indeed does not contain any char beyond Latin-1 charset.

Reply:
Thanks, maybe c++ user-defined literals could help here, make the code easier to read.